Tests/StreamChatTests/Workers/Background/ConnectionRecoveryHandler_Tests.swift

```diff
@@ -538,6 +541,7 @@ final class ConnectionRecoveryHandler_Tests: XCTestCase {
         XCTAssertCall(RetryStrategy_Spy.Signature.nextRetryDelay, on: mockRetryStrategy, times: 1)
         XCTAssertCall("incrementConsecutiveFailures()", on: mockRetryStrategy, times: 1)
         XCTAssertNotCall("syncLocalState(completion:)", on: mockChatClient.mockSyncRepository)
+        XCTAssertTrue(mockChatClient.mockExtensionLifecycle.receivedIsReceivingEvents == false)
```

Comment:
nit: Consider using `XCTAssert`

Reply:
Can't this be just:

```suggestion
        XCTAssertFalse(mockChatClient.mockExtensionLifecycle.receivedIsReceivingEvents)
```

🤔